SPECS/prometheus/prometheus.spec

```diff
@@ -0,0 +1,248 @@
+License:        ASL 2.0
+URL:            https://github.com/prometheus/prometheus
+#Source0:        https://github.com/prometheus/prometheus/archive/refs/tags/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+# unzip Source0
+# run 'make assets' in it
+# rm -rf web/ui/react-app
+# tar czvf web-ui-2.24.1.tar.gz web/ui
+Source10:       web-ui-%{version}.tar.gz
+Source1:        prometheus.service
+Source2:        prometheus
+Source3:        prometheus.yml
+Source4:        prometheus.conf
+Source5:        prometheus.logrotate
+Source6:        %{name}-%{version}-vendor.tar.gz
+
+# Debian patch for default settings
+Patch0:         02-Default_settings.patch
+
+BuildRequires:  systemd-rpm-macros
+BuildRequires:  golang
+Requires(pre):  /usr/bin/systemd-sysusers
+
+%description
+The Prometheus monitoring system and time series database
+
+%prep
+%autosetup -p1
+
+%build
+rm -rf web/ui
+tar xzvf %{SOURCE10}
+
+rm -rf vendor
+tar -xf %{SOURCE6} --no-same-owner
+
+export LDFLAGS="-X github.com/prometheus/common/version.Version=%{version} \
+                -X github.com/prometheus/common/version.Revision=%{commit_version} \
+                -X github.com/prometheus/common/version.Branch=master \
+                -X github.com/prometheus/common/version.BuildUser=mockbuild \
+                -X github.com/prometheus/common/version.BuildDate=%{builddate} "
+export BUILDTAGS="netgo builtinassets"
+for cmd in cmd/* ; do
+  go build -ldflags "$LDFLAGS" -mod=vendor -v -a -tags "$BUILDTAGS" -o $(basename $cmd) ./$cmd
+done
+
+%install
+install -m 0755 -vd                     %{buildroot}%{_bindir}
+install -m 0755 -vp prometheus          %{buildroot}%{_bindir}/
+install -m 0755 -vp promtool            %{buildroot}%{_bindir}/
+
+# Unit file
+install -m 0755 -vd                     %{buildroot}%{_unitdir}
+install -m 0644 -vp %{SOURCE1}          %{buildroot}%{_unitdir}/
+
+install -m 0755 -vd                     %{buildroot}%{_sysconfdir}
+install -m 0755 -vd                     %{buildroot}%{_sysconfdir}/prometheus
+install -m 0644 -vp %{SOURCE3}          %{buildroot}%{_sysconfdir}/prometheus/
+install -m 0755 -vd                     %{buildroot}%{_sysconfdir}/sysconfig
+install -m 0644 -vp %{SOURCE2}          %{buildroot}%{_sysconfdir}/sysconfig/
+install -m 0755 -vd                     %{buildroot}%{_sysconfdir}/logrotate.d
+install -m 0644 -vp %{SOURCE5}          %{buildroot}%{_sysconfdir}/logrotate.d/prometheus
+install -m 0755 -vd                     %{buildroot}%{_sysusersdir}
+install -m 0644 -vp %{SOURCE4}          %{buildroot}%{_sysusersdir}/
+
+mkdir -p %{buildroot}%{_sysconfdir}/prometheus/consoles
+mkdir -p %{buildroot}%{_sysconfdir}/prometheus/console_libraries
+mkdir -p %{buildroot}%{_sharedstatedir}/prometheus
+
+%pre
+%sysusers_create_package prometheus %{SOURCE4}
+
+%post
+%systemd_post prometheus.service
+
+%preun
+%systemd_preun prometheus.service
+
+%postun
+%systemd_postun_with_restart prometheus.service
+
+%if %{with check}
```

Comment:
You can remove this conditional around the check section, it's not necessary in our build system.